### PR TITLE
Fixed VRRP AH extraction in pcap2john

### PR DESCRIPTION
**Pull request by exploide**
Fixed the problem described in #4653.

Using scapy is more straight forward than dpkt.

I also cleaned up the imports a bit.

### CODE REVIEW

File: run/pcap2john.py [hunk 1]
```diff
@@ -1207,34 +1203,20 @@ def pcap_parser_ah(fname):
...
+            pkt = pkt['IP']
+            # zero mutable fields
+            pkt.tos = 0
+            pkt.flags = 0
+            pkt.chksum = 0
+            # store icv; zero icv in original packet
+            icv = pkt['AH'].icv
+            pkt['AH'].icv = bytearray(len(icv))
+            # print net-ah hash
+            print("$net-ah$0$%s$%s" % (hexlify(bytes(pkt)).decode('ascii'), hexlify(icv).decode('ascii')))
```
Comment (solardiz) [inline run/pcap2john.py:1198]: Looks like all other parsers use `sys.stdout.write`, not `print`. I don't really mind as long as this works across the same Python versions, although we could want to standardize on one or the other approach.

Comment (exploide) [reply run/pcap2john.py:1198]: `print()` should be available on more recent Python2 I think.

Agree that it should be standardized. I think `print` is a bit cleaner and more "pythonic" than `sys.stdout.write`. I changed the occurrences.

Comment (solardiz) [reply run/pcap2john.py:1198]: `print()` worked for me in a quick Python shell test with RHEL6's 2.6.6.

**Comment by exploide** [comment]
The other (non-AH) VRRP packets were also not correctly parsed with python3. Fixed that function, too.

It's amazing how simple the parsing becomes when using scapy.

**Pull request merged**